README.md

```diff
@@ -161,7 +158,7 @@ If you use Mava in your work, please cite the accompanying
 ```
 
 [Paper]: https://arxiv.org/pdf/2107.01460.pdf
-[quickstart]: https://github.com/instadeepai/Mava/blob/develop/Quickstart.ipynb
+[quickstart]: https://github.com/instadeepai/Mava/blob/develop//examples/Quickstart.ipynb
```

Comment:
Minor improvement (although it works without changing it):  `develop//examples/` -> `develop/examples/`